packages/editor/src/components/index.js

```diff
@@ -8,6 +8,7 @@ export { default as BlockEdit } from './block-edit';
 export { default as BlockFormatControls } from './block-format-controls';
 export { default as BlockNavigationDropdown } from './block-navigation/dropdown';
 export { default as BlockIcon } from './block-icon';
+export { default as BlockVerticalAlignmentToolbar } from './block-vertical-alignment-toolbar';
```

Comment:
@youknowriad, I assume this will have to be moved to `block-editor` package pretty soon, right?

Reply:
This would be picked up in a rebase.